### PR TITLE
Use correct env imports in build.

### PR DESCRIPTION
**Pull request by Hyperkid123**
### Changes
- configured babel to transform dependencies import based on `babel_env` property
  - cjs build bundle will use commojs imports
  - esm build bundle will use esm imports

This should help with the jest issues people are having in some platform apps. I have tested it with the vulnerability UI and if you use cjs imports, the jest issues are fixed and there is no need for transpiling some packages.

In the future, we should avoid directly referencing `cjs`/`esm` build of dependencies choose between them at build time.

There are some snapshot tests I will address and the skeleton size constant is a bit wonky but it should be fixed shortly.

@karelhala do we want to create some beta versions first?

### CODE REVIEW

**Comment by Hyperkid123** [comment]
@rvsia go away you are on vacation!

**Comment by Hyperkid123** [comment]
This time the build timed out :no_mouth:

**Comment by Hyperkid123** [comment]
Finally! IT IS :green_circle:

File: config/rollup-contants.js [hunk 1]
```diff
@@ -48,29 +48,34 @@ export const globals = {
     'react-redux': 'reactRedux'
 };
 
-export const rollupConfig = (external, plugins, globals, name, entries = [ globMapper('src/**/index.js') ], outputDir = './') => ([
-    ...[ 'esm', 'cjs' ].map(env => ({
+export const rollupConfig = (external, plugins, globals, name, entries = [ globMapper('src/**/index.js') ], outputDir = './') => {
+    console.warn(JSON.stringify(entries), process.env.FORMAT);
```
Comment (karelhala) [inline config/rollup-contants.js:52]: Do we really want to print out entries and format?

Comment (Hyperkid123) [reply config/rollup-contants.js:52]: no we don't I used it for debug :+1:

**Comment by Hyperkid123** [comment]
@karelhala I will wait for https://github.com/RedHatInsights/frontend-components/pull/702 to be merged and then rebase it. It should make it easier to review without the gazillion snapshot updates.

**Comment by karelhala** [comment]
Tag has been released, so we have a history point to go back to. Merging this!

**Review by karelhala — APPROVED** [review]

**Comment by Hyperkid123** [comment]
@AllenBW let me check. What version are you using? We have made some fixes since this was merged,

**Comment by AllenBW** [comment]
Hiiiii!! @Hyperkid123 just did n update, version 2.6.2 is the breaking one, 2.5.6 works just fine

**Comment by Hyperkid123** [comment]
Alright, let's get into it.

**Pull request merged**